package.json

```diff
@@ -6,18 +6,15 @@
   "license": "MIT",
   "scripts": {
     "prepare": "husky install",
-    "copy-views": "cp -R server/views dist/server/",
     "compile-assets": "grunt",
-    "watch-ts": "tsc -w",
-    "watch-views": "nodemon --watch server/views -e html,njk -x npm run copy-views",
-    "watch-node": "DEBUG=gov-starter-server* nodemon --watch dist/ dist/server.js | bunyan -o short",
-    "watch-assets": "grunt watch",
-    "build": "rm -rf dist && npm run compile-assets && tsc && npm run copy-views",
+    "watch-node": "ts-node-dev --respawn server.ts",
```

Comment:
Nodemon replaced with `ts-node-dev` for performance. Instead of watching for file changes `ts-node-dev` will watch the dependency graph of the application and automatically restart when changes are made. Under the hood it will run the application with `ts-node` avoiding the need for compilation.